build/BUILD

```diff
@@ -86,6 +86,12 @@ grep ^STABLE_BUILD_SCM_REVISION bazel-out/stable-status.txt \
     stamp = 1,
 )
 
+genrule(
+    name = "cni_package_version",
+    outs = ["cni_version"],
+    cmd = "echo 0.5.1 >$@",
```

Comment:
this feels a little weird, and I'm worried that this version is going to drift from what's used in the WORKSPACE.

I'm guessing the cni dependency doesn't have a version file we can use?

Reply:
we will be touching this dependency and in overall generation of cni package soon.
0.5.1 is the last usable for 1.7.x/1.8.x as single archive. For 1.9.x planned to be used CNI 0.6.x, and those are shipped as cni-tool and cni plugins in two different tarballs. So, we will anyway need to re-think this part and bits in WORKSPACE once CNI 0.6.x will arrive in master. :(